cdc/sink/dmlsink/mq/manager/kafka_manager.go

```diff
@@ -102,20 +102,12 @@ func (m *kafkaTopicManager) backgroundRefreshMeta(ctx context.Context) {
 			)
 			return
 		case <-m.metaRefreshTicker.C:
-			topicMetaList, err := m.getMetadataOfTopics(ctx)
 			// We ignore the error here, because the error may be caused by the
 			// network problem, and we can try to get the metadata next time.
-			if err != nil {
-				log.Warn("Get metadata of topics failed",
-					zap.String("namespace", m.changefeedID.Namespace),
-					zap.String("changefeed", m.changefeedID.ID),
-					zap.Error(err))
+			topicMetaList, _ := m.fetchNumPartitions(ctx)
```

Comment:
Please remove the comments above

Reply:
So , if an error happens, how to handle it?

Reply:
this log is redundant, since it's already logged in the `fetchNumPartitions` method